crates/task-impls/src/consensus.rs

```diff
@@ -1446,12 +1446,15 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                     return false;
                 };
 
+                // TODO: Put the signature received from the builder in the block header
                 let block_header = TYPES::BlockHeader::new(
                     state,
                     &consensus.instance_state,
                     &parent_leaf,
                     null_block_commitment,
                     metadata,
+                    0,
```

Comment:
Could we use a named constant instead of a magic number?

Reply:
https://github.com/EspressoSystems/HotShot/pull/2922/commits/deb3112ba4d674b16032da831337fcbcf8f2aac7